packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md

```diff
@@ -7,6 +7,148 @@ section: get-started
+- Multiple drop zones in the "next" drag and drop component.
+- [Editable table rows.](/components/table#editable-rows)
+- [Main section variations for the page component.](/components/page#main-section-variations)
+- [A masthead demo with horizontal navigation.](/components/masthead/react-demos)
+- [A basic description list demo.](/components/description-list/react-demos)
+
+#### New severity icons
+
+We created new severity icons, which can be used to communicate the severity of an error or issue with a data source that is linked to a UI element. These icons have been added to both our [Core](https://www.npmjs.com/package/@patternfly/patternfly) and [React](https://www.npmjs.com/package/@patternfly/react-icons) packages. They are also available for design use in [our Figma design kits.](https://www.figma.com/@patternfly) 
+
+![Severity icons within Figma menu](./img/severity-icons-figma.png)
+
+There are 6 icons that cover the following severity levels:
+- Undefined
+- None
+- Minor 
+- Moderate 
+- Important
+- Critical
+
+We are still working on writing design guidelines to document usage rules for these icons. Once these are completed and published, we will provide an update.
+
+#### New and updated React templates 
+
+We added 2 new react templates to help your migration efforts, including:
+- [Simple dropdown](/components/menus/dropdown/react-templates) 
+- [Typeahead select](/components/menus/select/react-templates) 
+
+We also updated some of our existing templates to address issues:
+- We fixed a bug that prevented initial selections from showing in simple select and checkbox select templates.
+- We fixed a bug that appeared after selections were cleared in a typeahead select.
+- We added additional customization support to our select templates.
+
+### Extensions updates
+#### Topology 
+
+We've expanded our Topology features to support a range of use cases, and fixed bugs that were found.
+
+**Pipelines**
+
+We added a few things to pipelines, including:
+
+- Custom status icons. 
+- `GroupLabelComponent`
+- Updates for expanded/collapsed groups.
+	- Support for focusing on the bounds of an expanded group.
+	- Adjusted the appearance of expanded group labels to match collapsed label styles.
+- Selectable task edges and selected task edge styling options.
+- Support for displaying nested, collapsed groups.
+
+**Group labels**
+
+- Added ability to center group labels on edges.
+- Added ability to display group labels on hover.
+
+**Control bar**
+
+- Added new expand/collapse all buttons.
+
+**Layouts**
+
+Updated the version of the dagre layout.
+
+**Task groups**
+
+- Added ability to display support status.
+
+We also went through all of our [Topology docs](/topology/about-topology) to tidy things up, expand on details, and make sure everything is consistent with other PatternFly docs. We also added documentation to provide guidance for using the Topology demo app. 
```

Comment:
We renamed the "getting started" page to "about topology" so _theoretically_ this link will work as long as that update is release with 5.4 (which I think it will be?)